operations/mimir-mixin/config.libsonnet

```
@@ -24,9 +24,12 @@
       ingester: '(ingester.*|cortex|mimir)',  // Match also custom and per-zone ingester deployments.
       distributor: '(distributor|cortex|mimir)',
       querier: '(querier.*|cortex|mimir)',  // Match also custom querier deployments.
+      ruler_querier: '(ruler-querier.*|cortex|mimir)',  // Match also custom querier deployments.
```

Monolithic deployment mode doesn't support it (because you will not have a dedicated pool of queriers for the ruler) so I would just support microservices deployment here.

```suggestion
      ruler_querier: '(ruler-querier.*)',  // Match also custom querier deployments.
```